lib/samson/process_utils.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Samson
+  module ProcessUtils
+    ATTRIBUTES = ['pid', 'ppid', 'gid', 'pcpu', 'user', 'start', 'args'].freeze
+    WHITELISTS = ['puma'].freeze
+
+    class << self
+      def ps_list
+        pipe = IO.popen("ps -eo #{ATTRIBUTES.join(',')} | grep -v #{WHITELISTS.join(',')}")
```

Comment:
- joining a grep with comma won't work
 - prefer a whitelist from ENV since every user will need a different whitelist (also allows fast adjustment)

Reply:
prefer filtering in ruby for more flexibility